File: l2-network/src/main/java/com/l2server/network/login/LoginClientConnection.java
package com.l2server.network.login;

import com.l2server.network.serverpackets.login.L2LoginServerPacket;

/**
 * Phoen-X on 04.07.2017.
 */
public interface LoginClientConnection {
    void send(L2LoginServerPacket packet);

    void disconnect();
}

File: l2-network/src/main/java/com/l2server/network/login/LoginClientConnection.kt
package com.l2server.network.login

import com.l2server.network.serverpackets.login.L2LoginServerPacket

/**
 * Phoen-X on 04.07.2017.
 */
interface LoginClientConnection {
    fun send(packet: L2LoginServerPacket)

    fun disconnect()
}
